zstacklib/zstacklib/utils/ovn.py

```
@@ -89,6 +89,30 @@ def getAllVfioPciNic():
+        if r != 0:
+            logger.warn("failed to get vnic port list for vm[uuid:%s], command[%s], output[%s]" % (vm_uuid, command, o))
+            return
+
+        vnic_names = o.splitlines()
+        for vnic_name in vnic_names:
+            vnic_name = vnic_name.strip()
+            if not vnic_name:
+                continue
+            logger.info('clean dpdk vnic port:%s from ovs' % vnic_name)
+            bash.bash_o(CtlBin + '--if-exists del-port {} {}'.format(brName, vnic_name))
+    except Exception as err:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)